src/index.html

```diff
@@ -1,101 +1,100 @@
 <!DOCTYPE html>
 <html lang="en-US">
   <head>
-    <title>Jeo</title>
+    <title>Jeopardy</title>
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,800" rel="stylesheet">
-
   </head>
   <body>
     <header>
       <h1>Jeopardy</h1>
     </header>
     <main>
       <section class='board'>
-        <article >
-                <h2 class='category-one'>category one</h2>
-            </article>
         <article>
-                <h2 class='category-two'>category two</h2>
-                </article>
+            <h2 class='category-one'>category one</h2>
+        </article>
         <article>
-                <h2 class='category-three'>category three</h2>
-            </article>
+            <h2 class='category-two'>category two</h2>
+        </article>
         <article>
-                <h2 class='category-four'>category four</h2>
-            </article>
-        <article id= 'clue' class='100'> 
-                <h3 class='100'>100</h3>
-            </article>
-        <article id= 'clue' class='100'> 
-                <h3>100</h3>
-            </article>
-        <article id= 'clue' class='100'> 
+            <h2 class='category-three'>category three</h2>
+        </article>
+        <article>
+            <h2 class='category-four'>category four</h2>
+        </article>
+        <article id= 'clue' class='col column-one-100' > 
```

Comment:
An `id` should be unique for the page/content. I would update your `id` of clue to a `class` of clue.